pom.xml

```diff
@@ -25,12 +25,12 @@
     <parquet.version>1.8.1</parquet.version>
     <!-- Edit the following line to configure the Hadoop (HDFS) version. -->
     <hadoop.version>2.6.0</hadoop.version>
-    <hadoop-bam.version>7.5.0</hadoop-bam.version>
+    <hadoop-bam.version>7.5.1-SNAPSHOT</hadoop-bam.version>
```

Comment:
7.6.0 was just released and has made it to Maven Central, so I'll push & rebase with the new version shortly